tests/conftest.py

```diff
@@ -0,0 +1,41 @@
+from pathlib import Path
+
+import pytest
+import numpy as np
+from PIL import Image
+from jina import Document, DocumentArray
+
+from executor import ImageHasher
+
+data_dir = (Path(__file__).parent / 'toy_data').absolute()
+
+
+@pytest.fixture(scope='function')
+def hasher():
+    return ImageHasher()
+
+
+@pytest.fixture(scope='function')
+def docs():
+    return DocumentArray(
```

Comment:
Is there a test verifying that dist(kid1, kid2) < dist(kid1, paprika1)?

Reply:
Not directly, but `test_match_quality` in `test_exec.py` checks the second search and matches it appropriately.